docs/settings.sh

```diff
@@ -52,7 +52,7 @@ DOCS_COMPONENT_PAGE_NAME="ansible"
 # Production S3 bucket that contains our production documentation repository.
 #
 # NOTE: You should never need to modify this setting!
-DOCS_BUCKET_PRODUCTION="docs-independent.confluent.io"
```

Comment:
this is throwing an error when I try to run the local build:

```
source ./settings.sh && source ~/.virtualenvs/cp-demo-docs/bin/activate
ERROR: DOCS_BUCKET_PRODUCTION must be set

```

Reply:
@kelvinL3 nevermind, user error. invoking using `make livehtml` works fine (as described in the [README](https://github.com/confluentinc/cp-demo/blob/6.1.1-post/docs/README.md)). thanks.